cmd/state/exec3/state_recon.go

```diff
@@ -15,381 +15,3 @@
 // along with Erigon. If not, see <http://www.gnu.org/licenses/>.
 
 package exec3
-
-import (
```

Comment:
Why only deleted all lines of code but not the file itself?

Reply:
i don't know